specifications/raml/api.raml

```diff
@@ -412,8 +347,10 @@ traits:
           example: "7ouMYWpwJ422jRcDASZB7P,4VqPOruhp5EdPBeR92t6lQ,2takcwOaAZWiXQijPHIx7B"
       responses:
         200:
-          body:
-            text/plain:
+        403:
```

Comment:
Is there a way to define a _global_ 403 status code? Note that this can be returned too when passing an invalid or expired access token to any endpoint.